src/main/java/com/epam/esm/entity/GiftCertificate.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.esm.entity;
+
+import java.math.BigDecimal;
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Set;
+
+public class GiftCertificate implements Identifiable {
+
+    private long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Duration duration;
+    private String creationDate;
+    private String lastUpdateDate;
```

Comment:
Let's store these date fields using Date class.

Reply:
Now it has ZonedDateTime class to store date